x-pack/plugins/cases/common/constants/index.ts

```diff
@@ -201,10 +201,9 @@ export const SEARCH_DEBOUNCE_MS = 500;
  * Local storage keys
  */
 export const LOCAL_STORAGE_KEYS = {
-  casesQueryParams: 'cases.list.queryParams',
-  casesFilterOptions: 'cases.list.filterOptions',
   casesTableColumns: 'cases.list.tableColumns',
   casesTableFiltersConfig: 'cases.list.tableFiltersConfig',
+  casesTableState: 'cases.list.state',
```

Comment:
There will be one state in local storage that will contain both the query params and the filers options.